src/megatron/energon/av/fastseek/containers/mpeg.py

```diff
@@ -184,8 +183,8 @@ def parse_atoms(file: BitsType) -> Generator[Atom, None, None]:
 def parse_mpeg(file: BitsType) -> dict[int, SortedList]:
     sync_samples = {}
     decode_timestamps = {}
-    presentation_time_offsets = defaultdict(lambda: defaultdict(lambda: 0))
-    start_offsets = defaultdict(lambda: 0)
+    presentation_time_offsets = {}
```

Comment:
This might cause a crash when processing CTTS atoms, I think that is where the defaultdict would previously have been initialized for a given `track_id` key